src/TTGamesExplorerRebirthLib/Formats/DDS/DDSImage.cs

```diff
@@ -35,7 +35,12 @@ public DDSImage(byte[] buffer)
                 Images[0] = bcDecoder.DecodeToImageRgba32(ddsFile);
             }
         }
-
+        ~DDSImage() 
+        {
+            Data = null;
+            Images = null;
+            GC.Collect();
+        }
```

Comment:
Since `Image` is `IDisposable` maybe we should do the same for `DDSImage` and then we can do `using DDSImage image = new();` to autodispose resources, the deconstructor here isn't needed.

Reply:
this entire block of code was just experimental that i forgot to remove, it does litterally nothing on a wider scale